src/js/VirtualFC.js

```diff
@@ -220,10 +222,11 @@ const VirtualFC = {
             "BEEPER MUTE",
             "READY",
             "LAP TIMER RESET",
+            "CHIRP",
         ];
```

Comment:
Remove the entries above and add them here instead:
```suggestion
        ];
    virtualFC.AUX_CONFIG.splice(virtualFC.AUX_CONFIG.indexOf("HORIZON") + 1, 0, "ALT_HOLD");
    virtualFC.AUX_CONFIG.splice(virtualFC.AUX_CONFIG.indexOf("CAMSTAB") + 1, 0, "POS_HOLD");
    virtualFC.AUX_CONFIG.push("CHIRP");
```

Reply:
Need to import import { API_VERSION_1_47 } from "./data_storage"; at the top of the file.

Reply:
in progress...

Reply:
considering 
```
$ grep -Iirn CONFIGURATOR.virtualApiVersion src/js/
src/js/VirtualFC.js:19:        virtualFC.CONFIG.apiVersion = CONFIGURATOR.virtualApiVersion;
src/js/serial_backend.js:124:                CONFIGURATOR.virtualApiVersion = PortHandler.portPicker.virtualMspVersion;
```
and 
![image](https://github.com/user-attachments/assets/d2df9760-0d9e-4a19-9a58-0892a9f4b846)
and 
![image](https://github.com/user-attachments/assets/61a684d7-9948-4c3c-84d0-c33ac44dd515)

I am unsure why it is not being set to 1.47 as of `9efd570e`.

Reply:
Also Vue does not remember the API version we has used.